src/Utilities.jl

```diff
@@ -101,11 +102,12 @@ function computesymbolsoverrange(
     numbersteps1d::Int;
     mass::Union{Operator,Nothing} = nothing,
     θ_min::Float64 = -π / 2,
+    θ_band::Float64 = 2π,
```

Comment:
; if we want these to really be keyword arguments as mentioned above
```suggestion
    mass::Union{Operator,Nothing} = nothing;
    θ_min::Float64 = -π / 2,
    θ_band::Float64 = 2π,
```